submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/service/AccNoServiceTest.kt

```diff
@@ -32,22 +32,57 @@ class AccNoServiceTest(
     @MockK private val privilegesService: IUserPrivilegesService
 ) {
     private val accNoPatternUtil: AccNoPatternUtil = AccNoPatternUtil()
-    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService)
+    private val testInstance = AccNoService(service, accNoPatternUtil, privilegesService, baseRelPath = "")
 
     @AfterEach
     fun afterEach() = clearAllMocks()
 
-    @ParameterizedTest(name = "prefix is {0} and numeric value is {1}")
-    @CsvSource(
-        "S-DIXA-AN-002, S-DIXA-AN-/002/S-DIXA-AN-002",
-        "S-BSST11, S-BSST/011/S-BSST11",
-        "S-DIXA-011, S-DIXA-/011/S-DIXA-011",
-        "1-AAA, 1-AAA/000/1-AAA",
-        "S-SCDT-EMBOJ-2019-103549, S-SCDT-EMBOJ-2019-/549/S-SCDT-EMBOJ-2019-103549",
-        "S-SCDT-EMBOR-2017-44445V1, S-SCDT-EMBOR-2017-44445V/001/S-SCDT-EMBOR-2017-44445V1"
-    )
-    fun getRelPath(value: String, expected: String) {
-        assertThat(testInstance.getRelPath(accNoPatternUtil.toAccNumber(value))).isEqualTo(expected)
+    @Nested
+    inner class GetRelPath {
+        private val testInstanceEmptyBase = AccNoService(service, accNoPatternUtil, privilegesService, baseRelPath = "")
+        private val testInstanceBaseWithSlash =
+            AccNoService(service, accNoPatternUtil, privilegesService, "baseRelPath/")
+        private val testInstanceBaseWithNoSlash =
+            AccNoService(service, accNoPatternUtil, privilegesService, "baseRelPath")
+
+        @ParameterizedTest(name = "prefix is {0} and numeric value is {1}")
+        @CsvSource(
+            "S-DIXA-AN-002, S-DIXA-AN-/002/S-DIXA-AN-002",
```

Comment:
lets simple include the `basePath` here in the parameters instead of creating two test classes. In that way we use single class.